src/anupSJun20/StringComparison.java

```diff
@@ -0,0 +1,37 @@
+package anupSJun20;
+/*Coding test 10 :
+
+Print minimum rotation required to compare both strings equality.
+
+Example 1  : 
+String input1 = "abcdef";
+String input2 = "defabc";
+output : 3
+
+Example 2  : 
+String input1 = "adbcdef"; -> push right
+fadbcde
+efadbcd
+defadbc
+String input2 = "defadbc";
+output : 3*/
+
+public class StringComparison {
+	
+	void findRotationRequired(String str1, String str2) {
+		int count = 0;
+		while(!str1.equalsIgnoreCase(str2)) {
```

Comment:
For positive scenario it looks super good. Just a suggestion. 

add one more condition in while loop to handle negative scenario as well otherwise if two string could not compare at all then it would be infinite loop. 

while(!str1.equalsIgnoreCase(str2) && count <= input1.length)